test/serializer/abstract/ResidualVoid.js

```diff
@@ -0,0 +1,5 @@
+y = global.__residual ? __residual("void", function() {
+  // A void residual function that has some side-effect that doesn't affect the heap
+  let res = 0;
```

Comment:
Yes just to illustrate a void function that does something but doesn't impact the heap